galaxy/templates/job-init-db.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "galaxy.fullname" . }}-init-db-{{ randAlphaNum 5 | lower }}
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+spec:
+  ttlSecondsAfterFinished: 10
+  template:
+    metadata:
+      generateName: "{{ include "galaxy.fullname" . }}-init-mounts-"
+      labels:
+        {{- include "galaxy.labels" . | nindent 8 }}
+      annotations:
+        checksum/galaxy_conf: {{ include (print $.Template.BasePath "/configs-galaxy.yaml") . | sha256sum }}
+        checksum/galaxy_rules: {{ include (print $.Template.BasePath "/configmap-galaxy-rules.yaml") . | sha256sum }}
+        checksum/galaxy_extras: {{ include (print $.Template.BasePath "/configmap-extra-files.yaml") . | sha256sum }}
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
+        - name: {{ .Chart.Name }}-init-postgres
+          image: alpine:3.7
+          command: ['sh', '-c', 'chown 101:101 {{ .Values.persistence.mountPath }}; until nc -z -w3 {{ template "galaxy-postgresql.fullname" . }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
+          volumeMounts: 
+            - name: galaxy-data
+              mountPath: /galaxy/server/database
+      containers:
+        - name: {{ .Chart.Name }}-db-init
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.initJob.createDatabase }}
+          command:
+            - 'sh'
+            - '-c'
+            - >
+              if (ls /galaxy/server/config/mutable/ | grep -q "db_init_done*");
+                then /galaxy/server/manage_db.sh upgrade && echo "Done" > /galaxy/server/config/mutable/db_init_done_{{.Release.Revision}};
+                else (/galaxy/server/create_db.sh && echo "Done" > /galaxy/server/config/mutable/db_init_done_{{.Release.Revision}};);
+              fi;
+          {{- else }}
+          command:
```

Comment:
Second `command`?

Reply:
One for `initJob.createDatabase=true` which expects to create the database on the first run and find signals from the create on the NFS, and another for `initJob.createDatabase=false` which always just runs migrations. The option only matters on startup, after startup they will both just run migrations, the second one would just know not to wait to create on the first run